explorer/src/providers/accounts/utils/metadataHelpers.ts

```diff
@@ -41,45 +41,88 @@ export async function getMetadata(
   cluster: Cluster,
   url: string
 ) {
-  let metadata;
+  try {
+    const accountInfo = await fetchMetadataFromPDA(pubkey, url);
+    if (accountInfo && accountInfo.data.length > 0) {
+      if (!isMetadataAccount(accountInfo)) return;
+
+      if (isMetadataV1Account(accountInfo)) {
+        const metadata = decodeMetadata(accountInfo.data);
+
+        if (
+          isValidHttpUrl(metadata.data.uri) &&
+          metadata.data.uri.indexOf("arweave") >= 0
```

Comment:
NFTs can use other options to host their URI than just arweave. I would suggest removing this as its also unnecessary.